OpenEdXMobile/src/main/java/org/edx/mobile/util/Version.java

```diff
@@ -142,4 +142,19 @@ public VersionParseException(final int location) {
             super("Token couldn't be parsed as a valid number.", location);
         }
     }
+
+    /**
+     * Compares this version to the specified version to determine if minor versions
+     * difference between both is greater than the or equal to the specified value.
```

Comment:
"app's current version" is not necessary here, it could be any version and can be compared to any other version.